spec/formatting.md

```diff
@@ -505,14 +505,19 @@ _Formatting_ is a mostly implementation-defined process,
 as it depends on the implementation's shape for resolved values
 and the result type of the formatting.
 
-Formatting errors MAY be emitted during _formatting_,
-as formatting is not necessarily defined on every resolved value.
-A formatter MAY provide a value to use in such a case instead of a _fallback value_.
-
-_Formatting_ MAY produce formatted messages with the following data types,
-as well as any others:
-
-- A single concatenated string.
+Resolved values cannot always be formatted by a given implementation.
+When such an error occurs during _formatting_, 
+an implementation SHOULD emit a _formatting error_ and produce a 
+_formatting fallback value_ for the _placeholder_ that produced the error.
+An implementation MAY substitute a value to use instead of a _falback value_.
+
+An implementation MAY use the value of an _expression_'s _operand_
```

Comment:
> implementation MAY use the value of an
implementation MAY use the type of an
?

Reply:
We don't have types. Implementations might have types. But that type would naturally be bound up in the operand's value (it would be the value's _type_ 😸)